openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/EntityPageInfo.tsx

```diff
@@ -385,53 +359,50 @@ const EntityPageInfo = ({
         entityType={(entityType as EntityType) ?? EntityType.TABLE}
         extra={
           <Space align="center" id="version-and-follow-section">
-            {!isUndefined(version) ? (
-              <>
-                {!isUndefined(isVersionSelected) ? (
-                  <Tooltip
-                    placement="bottom"
-                    title={
-                      <p className="tw-text-xs">
-                        {t('message.viewing-older-version')}
-                      </p>
-                    }
-                    trigger="hover">
-                    {getVersionButton(toString(version))}
-                  </Tooltip>
-                ) : (
-                  <>{getVersionButton(toString(version))}</>
-                )}
-              </>
-            ) : null}
+            {!isUndefined(version) && (
+              <VersionButton
+                className="m-l-xs tw-px-1.5"
+                selected={Boolean(isVersionSelected)}
+                version={toString(version)}
+                onClick={versionHandler}
+              />
+            )}
             {!isUndefined(isFollowing) ? (
-              <Button
-                className={classNames(
-                  'tw-border tw-border-primary tw-rounded',
-                  isFollowing ? 'tw-text-white' : 'tw-text-primary'
-                )}
-                data-testid="follow-button"
-                size="small"
+              <Dropdown.Button
+                icon={
+                  <Typography.Text
+                    className={classNames(
+                      isFollowing ? 'text-white' : 'text-primary'
+                    )}
+                    data-testid="follower-value">
+                    {followers}
+                  </Typography.Text>
+                }
+                menu={{
+                  items: [
+                    {
+                      key: 'followers',
+                      label: getFollowers,
+                    },
+                  ],
+                }}
+                trigger={['click']}
```

Comment:
Not sure why we need dropdown button here, are we showing something on click here?

Reply:
Yes list of followers on click of count.